tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -8347,7 +8347,7 @@ declare namespace ts.server {
         excludedFiles: ReadonlyArray<NormalizedPath>;
         private typeAcquisition;
         updateGraph(): boolean;
-        getExcludedFiles(): readonly NormalizedPath[];
+        getExcludedFiles(): ReadonlyArray<NormalizedPath>;
```

Comment:
I thought this had already been addressed in #29783?

Reply:
I un-addressed it to make the nightlies build again.